packages/sanity/src/core/form/inputs/CrossDatasetReferenceInput/ReferenceAutocomplete.tsx

```diff
@@ -57,7 +59,12 @@ export const ReferenceAutocomplete = forwardRef(function ReferenceAutocomplete(
               <Box padding={4}>
                 <Flex align="center" height="fill" justify="center">
                   <StyledText align="center" muted>
-                    No results for <strong>“{props.searchString}”</strong>
+                    <Translate
+                      t={t}
+                      i18nKey="inputs.reference.no-results-for-query"
+                      components={{SearchTerm: ({children}) => <strong>{children}</strong>}}
```

Comment:
should this be removed now that simple tags are allowed in the template string?

Reply:
Ah thanks - copied from previous work, can simplify this now.